packaging/nnstreamer.spec

```diff
@@ -42,9 +42,9 @@
 
 # Enable python if it's Tizen 5.0+
 %if 0%{tizen_version_major} >= 5
-%define		python_support 1
+%define		python3_support 1
 %else
-%define		python_support 0
+%define		python3_support 0
 %endif
```

Comment:
Because we are no longer supporting python2, you need to check if it's Tizen 6.5 to support python3. If it is under 6.5, do not enable python3.

Reply:
I checked that `Python3` is available and works on `Tizen-6.0` as well.